frontend/src/interfaces/gem.ts

```diff
@@ -0,0 +1,11 @@
+// TODO: replace with interface generated from protobuf?
+export interface Gem {
+  color: GemColor;
+}
+
+export interface GemColor {
+  // Hex code
+  selectionColor: String;
+  tintColor: String;
```

Comment:
Can probably use a type alias here instead so it's more descriptive

```
type HexCode = string;
```

Reply:
Nice, thanks!